controllers/servingruntime_controller_test.go

```diff
@@ -53,9 +53,9 @@ func waitForAndGetRuntimeDeployment(runtimeName string) *appsv1.Deployment {
 
 var _ = Describe("Sample Runtime", func() {
 	samplesToTest := []string{
-		"config/runtimes/mlserver-0.x.yaml",
-		"config/runtimes/triton-2.x.yaml",
-		"config/runtimes/ovms-1.x.yaml",
+		"controllers/testdata/mlserver-0.x.yaml",
+		"controllers/testdata/triton-2.x.yaml",
+		"controllers/testdata/ovms-1.x.yaml",
```

Comment:
I think it would be a lot cleaner for these to convert it from CSR to SR programmatically. Then we won't need to copy files around.

And it should be easy with manifestival, you can just have a top level function like
```go
func convertToServingRuntime(resource *unstructured.Unstructured) error {
	if resource.GetKind() == "ClusterServingRuntime" {
		resource.SetKind("ServingRuntime")
	}
	return nil
}
```
and then just add it to the existing Transform() call where each of the CRs are loaded, i.e.:
```go
		By("create the runtime")
		m, err = mf.ManifestFrom(mf.Path(filepath.Join("..", sampleFilename)))
		m.Client = mfc.NewClient(k8sClient)
		Expect(err).ToNot(HaveOccurred())
		m, err = m.Transform(convertToServingRuntime, mf.InjectNamespace(namespace))
		Expect(err).ToNot(HaveOccurred())
		// ...
```

Reply:
OK, it just feels like this is a bit more hacky/brittle.